src/redux/features/documentProducer/variables/variablesSelector.ts

```diff
@@ -183,3 +185,33 @@ export const selectVariablesWithValues = createCachedSelector(
     }
   }
 )((state: RootState, manifestId: number | string, documentId: number) => `${documentId}-${manifestId}`);
+
+export const selectDeliverableVariablesWithValues = createCachedSelector(
+  (state: RootState, deliverableId: number, projectId: number) =>
+    state.documentProducerDeliverableVariables[deliverableId],
+  (state: RootState, deliverableId: number, projectId: number) =>
+    state.documentProducerDeliverableVariableValues[deliverableCompositeKeyFn({ deliverableId, projectId })],
+  (variableList, valueList) => {
+    if (variableList?.data && valueList?.data) {
+      const variables = variableList.data;
+      const values = valueList.data;
+
+      let topLevelSectionPosition = 0;
+      const output = variableList.data.map((v: Variable) => {
+        if (v.type === 'Section' && v.renderHeading) {
```

Comment:
I am fairly confident this isn't needed in this context, there are no "sections" in deliverable questions. But I didn't want to cause more work in case there are functions / types that require some of the results of the `associateValues` function. We should look to remove this towards the end of the implementation.